packages/cli/src/commands/setup/deploy/providers/render.js

```diff
@@ -0,0 +1,114 @@
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from 'src/lib'
+
+const PROJECT_NAME = getPaths().base.match(/[^/|\\]+$/)[0]
+
+const RENDER_YAML = (database) => {
+  return `services:
+- type: web
+  name: ${PROJECT_NAME}-web
+  env: static
+  buildCommand: yarn rw deploy render web
+  staticPublishPath: ./web/dist
+  envVars:
+  - key: NODE_VERSION
+    value: 14
+  routes:
+  - type: rewrite
+    source: /.redwood/functions/*
+    destination: replace_me_with_api_url/*
+  - type: rewrite
+    source: /*
+    destination: /index.html
+
+- type: web
+  name: ${PROJECT_NAME}-api
+  env: node
+  buildCommand: yarn rw deploy render api
+  startCommand: cd api && yarn api-server --functions ./dist/functions
```

Comment:
We've recently changed this command:

```suggestion
  startCommand: cd api && yarn rw-api-server
```

Reply:
Related to comment by @dac09 (above in earlier outdated) related to https://github.com/redwoodjs/redwood/pull/2217

Reply:
After #2217 is merged then I will switch the startCommand to be `yarn rw serve api`